.travis/check-android-build

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
```

Comment:
Add `.sh` extension to this file to be consistent with other scripts?

Reply:
Will eliminating `.sh` be more feasible? And it's strongly preferred in `https://google.github.io/styleguide/shell.xml?showone=File_Extensions#File_Extensions`.

Reply:
I think the existing Travis scripts follow the examples on travis-ci.org, but I am fine with removing the `.sh` extension - we can do that in a separate PR.